broker/restapisrv.go

```diff
@@ -309,18 +309,6 @@ func (apisrv *RestApiSrv) DeleteLDEntity(w rest.ResponseWriter, r *rest.Request)
 	}
 }
 
-func (apisrv *RestApiSrv) DeleteLDAttribute(w rest.ResponseWriter, r *rest.Request) {
-	var eid = r.PathParam("eid")
-	var attr = r.PathParam("attr")
-
-	err := apisrv.broker.ldDeleteEntityAttribute(eid, attr)
-	if err == nil {
-		w.WriteHeader(204)
-	} else {
-		rest.Error(w, err.Error(), 404)
-	}
-}
-
 func (apisrv *RestApiSrv) LDGetEntity(w rest.ResponseWriter, r *rest.Request) {
```

Comment:
[Major] createdAt and modifiedAt should be hidden in attribute level.

Reply:
comment incorporated